### DISCUSSION TITLE
posts/use-pirate-with-guilty/

### DISCUSSION THREAD

**Discussion by giscus** [BYJRK/byjrk.github.io#15] [Announcements]
# posts/use-pirate-with-guilty/

国内盗版泛滥，正版意识缺乏。我们该如何看待使用盗版软件这一问题？

https://blog.coldwind.top/posts/use-pirate-with-guilty/

**Comment by Captain-Tom-CL** [comment]
确实,程序员用盗版软件打的是自己的脸

**Comment by DragonFlyInBlueSky** [comment]
软件太多了根本用不过来，用用免费的就行了，唉！身兼数职，码不完，根本码不完。

**Comment by abc35277** [comment]
这个咋说呢，和收入有直接关系。我收入万八千的时候我从没买过任何软件，现在收入高了我倒是也用过很多付费的软件但那都是我用过频率很高的东西。不过我现在确实更多的喜欢用社区版（如果能用的话）而不是破解的版本。在我国这个软件的定价和我国人的收入还是有差距的所以盗版比较猖獗。

**Reply by abc35277** [reply]
声明下我是支持力所能及的情况下使用正版的。
